lib/ui/page/home/page/chat/message_field/controller.dart

```diff
@@ -572,7 +572,28 @@ class MessageFieldController extends GetxController {
     if (file.size < maxAttachmentSize && _chatService != null) {
       try {
         var attachment = LocalAttachment(file, status: SendingStatus.sending);
-        attachments.add(MapEntry(GlobalKey(), attachment));
+
+        // If attachment is video or image insert it to end of img/video list and before documents, overwise simply add to end of attachments
```

Comment:
https://dart.dev/effective-dart/style#prefer-lines-80-characters-or-fewer

80 characters per line is a global standard limit: https://softwareengineering.stackexchange.com/questions/148677/why-is-80-characters-the-standard-limit-for-code-width

Plus don't forget, please, to end sentences with dots: https://dart.dev/effective-dart/documentation#do-format-comments-like-sentences